### PR TITLE
Add codeowners to AutoRest C# update

### PR DESCRIPTION
**Pull request by ShivangiReja**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -57,6 +57,9 @@
 # Reviewers to double check any API changes
 /sdk/**/api/                                    @KrzysztofCwalina @pakrym @tg-msft
 
+# Update AutoRest C#
+/eng/CodeGeneration.targets/                    @pakrym @ShivangiReja
```
Comment (pakrym) [inline .github/CODEOWNERS:61]: does it need a slash in the end after .targets?

**Review by pakrym — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (ShivangiReja) [reply .github/CODEOWNERS:61]: I'm not sure, I added a slash because I saw we added for others.

@chidozieononiwu Do we need a slash at the end?

Comment (pakrym) [reply .github/CODEOWNERS:61]: Others are directories and this one is a file, I don't think it's needed.

Comment (pakrym) [reply .github/CODEOWNERS:61]: see https://github.com/ShivangiReja/azure-sdk-for-net/blob/Add_CodeOwner/.github/CODEOWNERS#L69

Comment (ShivangiReja) [reply .github/CODEOWNERS]: Ah, right. Removed!

**Review by chidozieononiwu — APPROVED** [review]

**Pull request merged**